### PR TITLE
Fix #1292: Encounter Tabs Now Dropdown Upon Hover

### LINKED ISSUE #1292: No Hover Capability on Tabs on an Encounter Page
**GIF:**
tabhover

As you can see, the tabs may be clicked to reveal menu items but the menu items should reveal upon hover like everywhere else in the project.

**Replicate:**
Create a patient or have an existing one.
Make that patient active by selecting them through Patient/Client > Add New/Search and clicking the patient's name.
Create an encounter or have an existing one.
Select an encounter by clicking on an encounter from the dropdown of 'Past Encounter List'.
The tabs at the top are as referenced. 

**Comments:**
Just installed EHR with XAMPP on Windows 10 home. I am using Chrome.
Upon inspect on this page, I got these two errors but they shouldn't impact the tabs... unless I'm wrong.
image

### PR DESCRIPTION
**Pull request by jonathanzhang53**
## Description:
Changed onClick to onmouseover for menu items in encounters.

## GIF:
encounterfixfix

### CODE REVIEW

File: interface/patient_file/encounter/new_form.php [hunk 1]
```diff
@@ -258,7 +258,7 @@ function myGetRegistered($state="1", $limit="unlimited", $offset="0") {
       if($jid==0 || ($modid!=$modulerow['mod_id'])){
         if($modid!='')
         $StringEcho.= '</table></div></li>';
-      $StringEcho.= "<li><a href='JavaScript:void(0);' onClick=\"mopen('$DivId');\" >$new_category</a><div id='$DivId' ><table border='0' cellspacing='0' cellpadding='0'>";
+      $StringEcho.= "<li><a href='JavaScript:void(0);' onmouseover=\"mopen('$DivId');\" >$new_category</a><div id='$DivId' ><table border='0' cellspacing='0' cellpadding='0'>";
```
Comment (muarachmann) [inline interface/patient_file/encounter/new_form.php]: pretty simple to debug... 👍 nice one

**Comment by jonathanzhang53** [comment]
@muarachmann I added a mouseleave attribute that collapses the dropdown if the mouse leaves the tab or any of the tab's children.

File: interface/patient_file/encounter/new_form.php (hunk 1 above)
Comment (jonathanzhang53) [reply interface/patient_file/encounter/new_form.php]: added the mouseleave. onmouseleave doesn't work.

**Comment by jonathanzhang53** [comment]
@muarachmann Still haven't gotten past the force push. I don't know what I'm supposed to do. Git tells me to do 'git pull' first, but when I do that, it overwrites the commits that I already made.

When I use git commit --amend, I think I have to use git push -f because my local does not have commits ahead of my online repo.

**Comment by muarachmann** [comment]
@jonathanzhang53 why did you reomve mouseleave again?

**Comment by jonathanzhang53** [comment]
@muarachmann it wasn't working for me for some reason. I think the first time I tested it, I thought it worked but it really didn't (I was in a rush so I probably wasn't paying attention). So when I tested later, it actually didn't work so I removed it.

**Comment by muarachmann** [comment]
can handle this. ?

**Comment by jonathanzhang53** [comment]
@muarachmann sure thing.

**Comment by robbyoconnor** [comment]
I attempted to fix it but made a mess so reset it :)

**Comment by robbyoconnor** [comment]
I just squashed the commits

**Comment by robbyoconnor** [comment]
@jonathanzhang53 -- I re-pushed -- your commit added another commit -- you want to choose `s` or `squash` when you rebase.

**Comment by jonathanzhang53** [comment]
Yep I used to rebase purely locally I didn't know how to get a commit I made on github to appear on my local repo. Is it correct for me to use `git pull` to get that commit to show up or `git fetch origin`

**Comment by jonathanzhang53** [comment]
Had to push again @robbyoconnor @muarachmann because the merge conflict also overrode MigDinny's changes so I had to add them back manually

**Comment by robbyoconnor** [comment]
@jonathanzhang53 mine didn't...

**Comment by jonathanzhang53** [comment]
@robbyoconnor It should be fixed now. I just saw that migdinny's z-index change was showing up as 'deleted' but I just added it back in. :)

**Comment by robbyoconnor** [comment]
Squash your commits -- things look awkward now...

**Comment by robbyoconnor** [comment]
Fixed it.

**Comment by aethelwulffe** [comment]
OK.  I lied.  I just checked the onclick-mouseover changes and it is fine.

**Pull request merged**